packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -513,27 +521,29 @@ export const ReactEditor = {
     }
 
     if (!textNode) {
-      throw new Error(
-        `Cannot resolve a Slate point from DOM point: ${domPoint}`
-      )
+      return null as T extends true ? Point | null : Point
     }
 
     // COMPAT: If someone is clicking from one Slate editor into another,
     // the select event fires twice, once for the old editor's `element`
     // first, and then afterwards for the correct `element`. (2017/03/03)
     const slateNode = ReactEditor.toSlateNode(editor, textNode!)
     const path = ReactEditor.findPath(editor, slateNode)
-    return { path, offset }
+    return { path, offset } as T extends true ? Point | null : Point
   },
 
   /**
    * Find a Slate range from a DOM range or selection.
    */
 
-  toSlateRange(
+  toSlateRange<T extends boolean>(
     editor: ReactEditor,
-    domRange: DOMRange | DOMStaticRange | DOMSelection
-  ): Range {
+    domRange: DOMRange | DOMStaticRange | DOMSelection,
+    option: {
```

Comment:
This is inconsistent  -- using an option object versus a parameter for toSlatePoint above.   react-editor.ts has some existing use of this sort of pattern already and it uses an object called `options`.  Thus I would change both here and in  toSlatePoint to use an object called "`options`".

Reply:
And of course it is important for backward compatibility and consistency to not force options to be provided, like earlier in this file we find:
```
    options: { editable?: boolean } = {}
```